### PR TITLE
Update custom repo roles URL

### PR DESCRIPTION
**Pull request by exageraldo**
Update the API routes, from `/orgs/{org}/custom_roles` to `/orgs/{org}/custom-repository-roles`, and some documentation references.

Fixes: #2700

### CODE REVIEW

**Comment by gmlewis** [comment]
I'm surprised that GitHub did not make a version update for a breaking change like this. I will review this later tonight.

**Review by gmlewis — APPROVED** [review]
Thank you, @exageraldo !
LGTM.

Awaiting second LGTM+Approval from any other contributor to this repo before merging.

**Review by valbeat — APPROVED** [review]
LGTM!

**Comment by gmlewis** [comment]
> LGTM!

Thank you, @valbeat !
Merging.

**Pull request merged**
